<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Select Serial Port</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QComboBox" name="serialPortInfoListBox_3"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel_3">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="manufacturerLabel_3">
        <property name="text">
         <string>Manufacturer:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="serialNumberLabel_3">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="locationLabel_3">
        <property name="text">
         <string>Location:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="vidLabel_3">
        <property name="text">
         <string>Vendor ID:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="pidLabel_3">
        <property name="text">
         <string>Product ID:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="parametersBox">
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Select Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="baudRateLabel_3">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="baudRateBox_3"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataBitsLabel_3">
        <property name="text">
         <string>Data bits:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="dataBitsBox_3"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="parityLabel_3">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="parityBox_3"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="stopBitsLabel_3">
        <property name="text">
         <string>Stop bits:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="stopBitsBox_3"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="flowControlLabel_3">
        <property name="text">
         <string>Flow control:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="flowControlBox_3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="additionalOptionsGroupBox">
     <property name="font">
      <font>
       <family>Segoe UI Semibold</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Additional options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="localEchoCheckBox_3">
        <property name="text">
         <string>Local echo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton_3">
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
